test/distributed/org/apache/cassandra/distributed/test/SnapshotsTest.java

```diff
@@ -35,7 +35,10 @@
 import org.apache.cassandra.distributed.api.NodeToolResult;
 import org.apache.cassandra.distributed.shared.WithProperties;
 
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.apache.cassandra.db.ColumnFamilyStore.SNAPSHOT_DROP_PREFIX;
 import static org.apache.cassandra.distributed.shared.ClusterUtils.stopUnchecked;
+import static org.awaitility.Awaitility.await;
 
 public class SnapshotsTest extends TestBaseImpl
```

Comment:
do we also test (somewhere) that dropping of whole keyspace will create snapshots on individual tables?